gateway/1.5/modules/ROOT/pages/flex-gateway-renew-certificate.adoc

```diff
@@ -0,0 +1,340 @@
+=== Renew Registration with Connected App Credentials
+
+Flex Gateway includes the following CLI command to renew registration using connected app credentials:
+
+[source,ssh]
+----
+flexctl registration renew --client-id=<your-client-id> --client-secret=<your-client-secret> \
+ <path-to-registration-file>
+----
+
+If you do not have a username or password for Anypoint Platform, you can configure connected apps via Anypoint Access Management:
+
+* xref:access-management::connected-apps-developers.adoc#create-a-connected-app[Configure a Connected App]
+** Include the following scopes:
+*** Read Servers
+*** Manage Servers
+*** View Organization
+** Save the *Id* and *Secret* of the Connected app you configure, then use these credentials in the `flexctl registration renew` command.
+
+Replace `<path-to-registration-file>` with the path and filename of the existing `registration.yaml` file. You can optionally output the new registration file to a directory by using the `--output-directory` flag. The `renew` command overwrites the existing registration file if you specify the file's directory as the output directory.
+
+IMPORTANT: Create a new directory called `flex-renew-registration` (or similar) and then run the `renew` command in this new directory. The command creates registration files in this location.
+
+For Flex Gateway running in a container, invoke `renew` as part of the container `run` command:
+
+[tabs]
+====
+Docker::
++
+[source,ssh]
+----
+docker run --entrypoint flexctl \
+-v "$(pwd)":/renew \
+-v <path-to-registration-directory>:/registration \
+-u $UID mulesoft/flex-gateway \
+registration renew \
+--client-id=<your-client-id> \
+--client-secret=<your-client-secret> \
+--output-directory=/renew \
+/registration/<registration-filename>
+----
+
+Podman::
++
+[source,ssh]
+----
+podman run --entrypoint flexctl --userns=keep-id \
+-v "$(pwd)":/renew:Z \
+-v <path-to-registration-directory>:/registration:Z \
+-u $UID docker.io/mulesoft/flex-gateway \
+registration renew \
+--client-id=<your-client-id> \
+--client-secret=<your-client-secret> \
+--output-directory=/renew \
+/registration/<registration-filename>
+----
+====
+
+Invoking the command results in the following output:
+
+[source,text]
+----
+Starting registration renewal, please be patient.
+Registration renewal completed, the configuration files were written in directory ".".
+For security, modify the file permissions to restrict production scenario access to the user running flex.
+
+Side effect: the following files were written in the current directory
+registration.yaml
+----
+
+[[inspection-cli-overview]]
+== Registration Inspection
+
+Flex Gateway includes the following CLI command to inspect registration:
+
+----
+flexctl registration inspect
+----
+
+The command inspects your Flex Gateway registration, extracting information you use to verify whether the renewal command was successful.
+
+Invoke the following `help` command for information about usage:
```

Comment:
New task?

Reply:
See note above about the "help" command not being a part of the overall steps.

Reply:
Fixed! I moved the help instructions to a level 2 section at the bottom.